bot01.user.js

```diff
@@ -211,6 +211,25 @@
         threadId = uw.threadIds[account]; // ID of thread that the bot should subscribe
         townId = uw.townIds[account]; // ID of the bot's town - it needs to own it all the time
 		const olympusId = uw.olympusId;
+        let body = {"types":[{"type":"map","param":{"x":8,"y":7}},{"type":"bar"},{"type":"backbone"}],"town_id":townId,"nl_init":false};
+        uw.gpAjax.ajaxPost('data', 'get', body, true, {
+                success: function(layout, resp, succ, t_token) {
+                    let server_time = getTimestampAndSetZone(resp);    
+                }
+            });
+    }
+
+    function getTimestampAndSetZone(resp) {
+        timestamp = resp.json.map.data.data.data[0].chunk.timestamp * 1000;
+        now = Date.now();
+        now = new Date(now);
+        timestamp = new Date(timestamp);
```

Comment:
i suggest:
const server_time = new Date(timestamp);
because nnow it looks messy